src/logic/programmable-function-logic.ts

```diff
@@ -26,20 +27,16 @@ export class ProgrammableFunctionLogic extends WatchableFunctionLogic {
   protected answerByArgs: { answer: ProgrammedAnswer; args: unknown[] }[] = [];
 
   constructor(
+    private address: Address,
```

Comment:
I belive this address is not being used

Reply:
Good catch, thanks! Fixed in c4177d30174c731ad162148ce94a82221cbe0d22